lib/modules/hosts/youtube.js

```diff
@@ -11,47 +11,50 @@ export default {
 			(/^https?:\/\/(?:youtu\.be|(?:www\.|m\.)?youtube\.com)\/?(?:watch|embed)?(?:.*?v=|v\/|\/)([\w-]+)/i).exec(href);
 	},
 	handleLink(elem, [, hash]) {
-		// Check url for timecode e.g t=1h23m15s
-		const starttimeRe = /[&\?](?:t=|start=)([0-9hms]+)/i;
-		const starttimeResult = starttimeRe.exec(elem.href);
-		// End time is always in integer format, no need to parse
-		const endtimeRe = /[&\?]end=([0-9]+)/i;
-		const endtimeResult = endtimeRe.exec(elem.href);
-		// Finds playlist id if included
-		const playlistRe = /[&\?]list=([\w\-_]+)/i;
-		const playlistResult = playlistRe.exec(elem.href);
+		let src = `https://www.youtube.com/embed/${hash}?enablejsapi=1&enablecastapi=1`;
 
-		let starttime = 0;
-		if (starttimeResult) {
-			const timeBlocks = { h: 3600, m: 60, s: 1 };
-			const timeRe = /[0-9]+[hms]/ig;
-			// Get each segment e.g. 8m and calculate its value in seconds
-			const timeMatch = starttimeResult[1].match(timeRe);
-
-			if (timeMatch) {
-				timeMatch.forEach(ts => {
-					const unit = timeBlocks[ts.charAt(ts.length - 1)];
-					const amount = parseInt(ts.slice(0, -1), 10);
-					// Add each unit to starttime
-					starttime += unit * amount;
-				});
-			} else {
-				// support direct timestamp e.g. t=200
-				starttime = parseInt(starttimeResult[1], 10);
-				if (isNaN(starttime)) starttime = 0;
+		const findParamsRe = /[\?&]([\w-]+)=([\w-]+)/g;
+		let paramResults;
+		while ((paramResults = findParamsRe.exec(elem.href)) !== null) {
```

Comment:
`getUrlParams` can be used to do this; you can import it as `import { getUrlParams } from '../../utils';`